packages/toast/src/toast.class.tsx

```diff
@@ -1,6 +1,6 @@
 import { isBrowser } from "@chakra-ui/utils"
 import * as React from "react"
-import { render } from "react-dom"
+import * as ReactDom from "react-dom"
```

Comment:
This won't work, because we need to `import { createRoot } from 'react-dom/client'`
But then it is React 18 specific and we cannot support React 17 🤔 

I think we need to move to a toast implementation with `createPortal` immediately to support React 18.

Reply:
That's true. Let's chat about this in the meeting today